data/core/terrain-graphics/deprecated-base.cfg

```diff
@@ -1,6 +1,9 @@
 #textdomain wesnoth
 
 #toplevel macros to handle base tiles
+# These macros are legacy code, the entire file can be retired
+
+#deprecated 2 1.19.0 Any terrain macro with a TERRAIN_BASE_* name pattern is being retired and uses deprecated internal macros
```

Comment:
This is not the correct way to deprecate these. It will result in the deprecation message being shown _always_. You'll need to copy-paste this line into each macro individually.

Basically, this deprecation message is shown when someone includes this file as `{core/terrain-graphics/deprecated-base.cfg}`, but the core _main.cfg does this, so it will always show.

Reply:
I know, thanks.